google/cloud/completion_queue.h

```diff
@@ -125,13 +136,9 @@ class CompletionQueue {
       typename std::enable_if<Sig::value, int>::type = 0>
   future<StatusOr<Response>> MakeUnaryRpc(
       AsyncCallType async_call, Request const& request,
-      std::unique_ptr<grpc::ClientContext> context) {
-    auto op =
-        std::make_shared<internal::AsyncUnaryRpcFuture<Request, Response>>();
-    impl_->StartOperation(op, [&](void* tag) {
-      op->Start(async_call, std::move(context), request, impl_->cq(), tag);
-    });
-    return op->GetFuture();
+      std::shared_ptr<grpc::ClientContext> context) {
```

Comment:
Did you mean to make this a `std::shared_ptr<>`?  Seems like an unrelated change.

Reply:
No, good catch. That was an unintentional change.

(It only now dawns on me that changing this method to accept a `std::shared_ptr<>` is not a breaking change. So I was not blocked on it. Oh well, we want this change anyway.)